extensions/wikia/PortableInfobox/js/PortableInfobox.js

```diff
@@ -33,9 +33,13 @@
 
 			$collapsibleGroups.each( function( index, group ) {
 				var $group = $collapsibleGroups.eq(index),
-					$header = $group.find('.pi-header');
+					$header = $group.find('.pi-header'),
+					title = $group.hasClass('pi-collapse-closed') ? 'Open' : 'Close';
```

Comment:
Great point. This is a sitewide functionality and as such needs to be translatable